src/main/java/de/neuland/jade4j/lexer/token/Attribute.java

```diff
@@ -1,13 +1,13 @@
 package de.neuland.jade4j.lexer.token;
 
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 import de.neuland.jade4j.parser.node.ExpressionString;
 
 public class Attribute extends Token {
 
-	Map<String, Object> attributes = new HashMap<String, Object>();
+	Map<String, Object> attributes = new LinkedHashMap<String, Object>();
```

Comment:
@waldemar-schneider Seems good! I would only suggest for you to reduce the accessibility of `attributes` by changing it from `Map<String, Object> attributes` to `private Map<String, Object> attributes`.